OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -543,9 +543,10 @@ public class LobbyOption
 		public readonly bool IsLocked;
 		public readonly bool IsVisible;
 		public readonly int DisplayOrder;
+		public readonly string Category;
 
 		public LobbyOption(string id, string name, string description, bool visible, int displayorder,
-			IReadOnlyDictionary<string, string> values, string defaultValue, bool locked)
+			IReadOnlyDictionary<string, string> values, string defaultValue, bool locked, string category = null)
```

Comment:
Sorry for the scope creep... but can we please remove the default value here, and instead add category fields (defaulting to null of course) for the current lobby options?

This will mostly be a copy-paste job. You can use this as a template for the option description:
```csharp
[Desc("Options category in which to display the <e.g. 'developer mode checkbox' or 'DefaultCash option'> in the lobby.")]
```